gosushi_client/src/App.scss

```diff
@@ -3,34 +3,33 @@
 }
 
 .App-logo {
-  background-color: black;
+  background-color: #8d5820;
   width: 100%;
   padding: 10vh 0 10vh 0;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.App-content {
+  background-color: #f6edce;
+  height: 100vh;
```

Comment:
Adding this kills the header in Host-Game page - should probably be reconsidered